packages/core/src/awsService/ec2/model.ts

```diff
@@ -271,38 +286,74 @@ export class Ec2Connecter implements vscode.Disposable {
         return logger
     }
 
-    public async configureSshKeys(selection: Ec2Selection, remoteUser: string): Promise<SshKeyPair> {
+    public async configureSshKeys(selection: Ec2Selection, remoteUser: RemoteUser): Promise<SshKeyPair> {
         const keyPair = await SshKeyPair.getSshKeyPair(`aws-ec2-key`, 30000)
         await this.sendSshKeyToInstance(selection, keyPair, remoteUser)
         return keyPair
     }
 
+    public async attemptToCleanKeys(
```

Comment:
"try" is a common convention that fits here. the body of the function isn't obvious, so worth a docstring too.

```suggestion
    /** Removes public key(s) that we added to the remote ~/.ssh/authorized_keys file. */
    public async tryCleanKeys(
```